packages/tizen_notification/lib/src/types.dart

```diff
@@ -2,102 +2,189 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: public_member_api_docs
+import 'package:tizen_app_control/tizen_app_control.dart';
 
-import 'enums.dart';
+export 'package:tizen_app_control/tizen_app_control.dart';
 
-/// Properties that configure how the system handles notification at
-/// certain scenarios.
-class Property {
-  /// Display only SIM card inserted.
-  static const int displayOnlySimMode = 1 << 0;
+/// How the system handles the notification in certain scenarios.
+class NotificationProperty {
+  /// Display only if a SIM card is inserted.
+  static const int onlySimMode = 1 << 0;
 
-  /// Disable application launch when notification is selected.
+  /// Do not perform any operation when the notification is clicked.
   static const int disableAppLaunch = 1 << 1;
 
-  /// Disable auto delete when notification is selected.
+  /// Do not dismiss the notification when clicked.
   static const int disableAutoDelete = 1 << 2;
 
-  /// Delete notification when device is rebooted.
-  static const int volatileDisplay = 1 << 8;
+  /// Dismiss the notification when the device is rebooted.
+  static const int volatile = 1 << 8;
 }
 
-/// The destination app that displays notification.
-class DisplayApplist {
-  /// Notification Tray(Quickpanel).
+/// Where and how the notification should be presented.
+class NotificationStyle {
+  /// Display in the notification area of the quick panel.
   static const int tray = 1 << 0;
 
-  /// Ticker notification.
-  static const int ticker = 1 << 1;
-
-  /// Lock screen.
+  /// Display in the lock screen.
   static const int lock = 1 << 2;
 
-  /// Indicator.
-  static const int indicator = 1 << 3;
+  /// Display in the indicator area (the top of the screen).
+  static const int indicator = 1 << 1 | 1 << 3;
```

Comment:
`NOTIFICATION_DISPLAY_APP_TICKER` and `NOTIFICATION_DISPLAY_APP_INDICATOR` do not work in the way they are [documented](https://docs.tizen.org/application/native/guides/notification/notifications/#indicator-notifications) and now they are a single property.
- `NOTIFICATION_DISPLAY_APP_INDICATOR` has no effect on mobile (only `NOTIFICATION_DISPLAY_APP_TICKER` works in an unexpected way).
- `NOTIFICATION_DISPLAY_APP_TICKER` is effectively the same as `NOTIFICATION_DISPLAY_APP_INDICATOR` on RPi.